test/lib/prisma.test.ts

```diff
@@ -0,0 +1,101 @@
+
+import {it, expect} from '@jest/globals';
+import {whereAndSelect} from "@lib/prisma";
```

Comment:
Codacy has a fix for the issue: [Replace `whereAndSelect` with `·whereAndSelect·`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7713215)
```suggestion
import { whereAndSelect } from "@lib/prisma";
```